packages/node/src/generators/application/application.ts

```diff
@@ -99,6 +101,7 @@ function getEsBuildConfig(
       ),
       // Use CJS for Node apps for widest compatibility.
       format: ['cjs'],
+      bundle: false,
```

Comment:
Would this change turn off bundling for all node servers?

Reply:
oops, bad rebase

Reply:
👍🏿